packages/report-server/reports.hjson

```diff
@@ -1,262 +0,0 @@
-{
```

Comment:
I've got a copy of this file if anyone wants it to copy/learn different report configs from